RUN3/AliAnalysisTaskAO2Dconverter.cxx

```diff
@@ -490,7 +493,11 @@ void AliAnalysisTaskAO2Dconverter::UserExec(Option_t *)
 
   vtx.fNentries[kEvents] = 1;  // one entry per vertex
   vtx.fRunNumber = fESD->GetRunNumber();
-  vtx.fGlobalBC = GetEventIdAsLong(fESD->GetHeader());
+  ULong64_t evtid = GetEventIdAsLong(fESD->GetHeader());
+  if(!evtid){
+    evtid = (ULong64_t(fESD->GetTimeStamp())<<32) + ULong64_t((fESD->GetNumberOfTPCClusters()<<5)|(fESD->GetNumberOfTPCTracks()));
```

Comment:
Do we really have to encode there the number of TPC clusters and tracks? How is the time stamp created in MC?

Reply:
This is Ruben's idea. The idea is to "randomize" it for the case that two events are produced at the same time on different Grid nodes.

Reply:
It seems that in the MC simulation we are using all the time stamps are 0. To get different behaviour, one has to set the flag fUseTimeStampFromCDB>0. What if we set it simply to Random covering some period in 2021?

Reply:
Or use the current time when the conversion job is running? We could use fESD->GetTimeStamp and fall back to the current time if that is 0...